topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -227,16 +240,22 @@ Our objective is to find which exon contains the most SNPs. Therefore we have to
 {: .comment}
 
 > ### {% icon hands_on %} Hands-on: Finding Exons
+> 
+> To find intersection we will be using `intersect intervals` tool from [BEDTools](https://bedtools.readthedocs.io/en/latest/content/overview.html) package. 
 >
-> 1. {% tool [Join](toolshed.g2.bx.psu.edu/repos/devteam/join/gops_join_1/1.0.0) %}   the intervals of two datasets side-by-side:
+> 1. {% tool [bedtools intersect intervals](toolshed.g2.bx.psu.edu/repos/iuc/bedtools/bedtools_intersectbed/2.30.0) %} the intervals of two datasets side-by-side:
 >
 >    Enter the word `join` in the search bar of the tool panel, and select the
 >    tool named `Join - the intervals of two datasets side-by-side`
 >
->    - *"Join"*: Select `Exons`
->    - *"With"*: Select `SNPs`
->    - *"with min overlap"*: `1`
->    - *"Return"*: `Only records that are joined (INNER JOIN)`, which means that only matches are included in the output (i.e. only exons with SNPs in it and only SNPs that fall in exons)
+>    - *"File A to intersect with B"*: Select `Exons`
+>    - *"Combined or separate output files"*: Select `One output file per 'input B' file"`
+>    - *"File B to intersect with A"*: `SNPs`
+>    - *"What should be written to the output file?"*: `Write the original entry in B for each overlap...`, which means that only matches are included in the output (i.e. only exons with SNPs in it and only SNPs that fall in exons)
+>
+>  The interface of the tool should look like this:
+>
+>    ![Contents of the `Join` output dataset](../../images/101_bed_intersect_interface.png)
```

Comment:
I'm -0 on this, it's now an image that we need to keep updated (and as you've seen from the other UI screenshots you replaced, we already weren't able to keep those updated). It's a 101 tutorial so I understand the motivation, maybe another reviewer wants to weigh in here.

Reply:
I am VERY much for this. In case of complex interfaces it is ESSENTIAL to have pictures.

Reply:
moreover, we need to ultimately to auto generate interface images for all tools in a tutorial. Personally, I become dizzy from looking at lists with icons, but just one glance at an image is sufficient to get the point

Reply:
I am 100% in support of it, if they're autogenerated! Then we could even make it sort of a preference for the user, do you want the text list, or the image.

The work Oleg did was fantastic, we need to scale this up. How do we do that @nekrut? Seems like a GOAT priority but I think nick is completely busy with the hub2.

Reply:
Well ... for now let keep the figure in. Nick will soon be done with hub2

Reply:
If that's the trade: screenshots included temporarily, in exchange for Nick's time after the hub to automate, ok! I agree! :100: Issue resolved.